.github/workflows/nightly.yml

```diff
@@ -31,4 +31,12 @@ jobs:
       flink_version: ${{ matrix.flink }}
       flink_url: https://s3.amazonaws.com/flink-nightly/flink-${{ matrix.flink }}-bin-scala_2.12.tgz
       cache_flink_binary: false
-    secrets: inherit
\ No newline at end of file
+    secrets: inherit
+
+  python_test:
+    strategy:
+      matrix:
```

Comment:
Wonder if we should consider using a similar matrix for python versions

Reply:
The python matrix is handled within the step. This is a good idea, but will require a change to the `flink-connector-shared-utils`, will create a follow up Jira